homeassistant/components/cover/mqtt.py

```diff
@@ -44,11 +44,17 @@
     vol.Optional(CONF_STATE_CLOSED, default=STATE_CLOSED): cv.string,
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
     vol.Optional(CONF_RETAIN, default=DEFAULT_RETAIN): cv.boolean,
+    vol.Optional(CONF_VALUE_TEMPLATE): cv.template,
```

Comment:
The mqtt platform rw template already has this in its schema. I'll remove this in my PR to fix retain for mqtt cover.